wiki/Gameplay/Game_modifier/Perfect/en.md

```diff
@@ -40,7 +40,8 @@ Any of the following acts **will not cause** the Perfect mod to restart the beat
 - Obtaining a `300`
 - Completing a spinner
 - Obtaining a Rainbow 300/MAX score ([osu!mania](/wiki/Game_mode/osu!mania))
-- Obtaining a 300 score value (osu!mania)
+- Obtaining a 300 score value ([osu!mania](/wiki/Game_mode/osu!mania))
```

Comment:
isn't this covered by the first item in this list? (line 40)

Reply:
Fixed, thanks!